textproc/xsv/Portfile

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cargo 1.0
+PortGroup           github 1.0
+
+github.setup        BurntSushi xsv 0.13.0
+categories          textproc
+description         A fast CSV command line toolkit written in Rust
+long_description    xsv is a command line program for indexing, slicing, \
+                    analyzing, splitting and joining CSV files.
+license             MIT
+platforms           darwin
+maintainers         {@lperry perry} \
+                    openmaintainer
```

Comment:
Usually, stylistically, we put these on one line. It makes grepping easier.

Reply:
Thanks, will collapse these lines the next time I update the port.

Reply:
Is there a good reason not to do it now? Force pushes are inexpensive. :)

Reply:
My understanding of this is that once you've committed something to master, you should not force-push thereafter. It can cause all kinds of unpredictable trouble if anything has come along after you and modified that Portfile, or something that depends on that commit otherwise.

Force-pushing is fine, desirable, in a PR to modify it prior to being committed.

Do I have this wrong? I thought it was more or less a rule of git-fu.

Reply:
Ah, I thought it hadn't been committed to master. My mistake.